src/biokbase/narrative/jobs/jobcomm.py

```diff
@@ -112,6 +114,11 @@ def cell_id_list(self):
             return self.rq_data[PARAM["CELL_ID_LIST"]]
         raise JobRequestException(CELLS_NOT_PROVIDED_ERR)
 
+    @property
+    def ts(self):
+        """This param is completely optional"""
```

Comment:
@n1mus is going to write a better description for this param in a forthcoming PR 😁